connector/mqtt/locust/src/ejbca/certificate.py

```diff
@@ -20,20 +20,10 @@ def __init__(self, thing_id):
         self.jwt = RedisClient().get_jwt()
 
         self.c_name = thing_id
```

Comment:
The `thing_id` no longer needs to contain the tenant, the function `Utils.create_thing_id(tenant, device_id)` is no longer needed.